administrator/language/en-GB/com_users.ini

```
@@ -59,10 +57,29 @@ COM_USERS_CONFIG_FIELD_SUBJECT_PREFIX_LABEL="Subject Prefix"
 COM_USERS_CONFIG_FIELD_USERACTIVATION_LABEL="New User Account Activation"
 COM_USERS_CONFIG_FIELD_USERACTIVATION_OPTION_ADMINACTIVATION="Administrator"
 COM_USERS_CONFIG_FIELD_USERACTIVATION_OPTION_SELFACTIVATION="Self"
+COM_USERS_CONFIG_FORCEMFAUSERGROUPS_DESC="Any user who belongs in <em>any</em> of the following user groups will be required to enable Multi-factor Authentication before being able to use the site."
+COM_USERS_CONFIG_FORCEMFAUSERGROUPS_LABEL="Enforce Multi-factor Authentication"
+COM_USERS_CONFIG_FRONTEND_CAPTIVE_TEMPLATE_DESC="Choose the front-end template style to use in the Multi-factor Authentication page. Select “Use Global” to use the default site template style."
```

```suggestion
COM_USERS_CONFIG_FRONTEND_CAPTIVE_TEMPLATE_DESC="Choose the frontend template style to use in the Multi-factor Authentication page. Select “Use Global” to use the default site template style."
```

As per the style guide. "One word not hyphenated or two words."